.github/workflows/sonarqube.yml

```
@@ -36,47 +36,25 @@ jobs:
           java-version: ${{ env.JAVA_VERSION }}
           distribution: 'microsoft'
 
-      - name: Read .terraform-version
-        run: |
-          DOTFILE_VERSION=$(cat terraform/.terraform-version)
-          echo "TF_VERSION=$DOTFILE_VERSION" >> $GITHUB_ENV
-
-      - name: Set up Terraform ${{ env.TF_VERSION }}
-        uses: hashicorp/setup-terraform@v3
-        with:
-          terraform_version: ${{ env.TF_VERSION }}
-
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v3
-
       - name: Cache SonarCloud scanner
         id: cache-sonar-scanner
         uses: actions/cache@v4
         with:
-          path: .\.sonar\scanner
+          path: ~/.sonar
           key: ${{ runner.os }}-sonar-scanner
           restore-keys: ${{ runner.os }}-sonar-scanner
 
       - name: Install SonarCloud scanner
         run: |
-          dotnet tool install --global dotnet-sonarscanner --version 8.0.3
-          dotnet tool install --global dotnet-reportgenerator-globaltool --version 5.4.4
-          dotnet tool restore
+          dotnet tool install --global dotnet-sonarscanner --version 10.1.0
 
       - name: Begin SonarQube Scanner
-        run: dotnet-sonarscanner begin /d:sonar.qualitygate.wait=true /k:"DFE-Digital_find-information-about-academies-and-trusts" /o:"dfe-digital" /d:sonar.token="${{ secrets.SONAR_TOKEN }}" /d:sonar.host.url="https://sonarcloud.io" /d:sonar.cs.vscoveragexml.reportsPaths=coverage.xml /d:sonar.scanner.skipJreProvisioning=true /d:sonar.buildString=${{ env.LAST_COMMIT_SHA }}
+        run: dotnet-sonarscanner begin /d:sonar.qualitygate.wait=true /k:"DFE-Digital_find-information-about-academies-and-trusts" /o:"dfe-digital" /d:sonar.token="${{ secrets.SONAR_TOKEN }}" /d:sonar.host.url="https://sonarcloud.io" /d:sonar.scanner.skipJreProvisioning=true /d:sonar.buildString=${{ env.LAST_COMMIT_SHA }}
 
       - name: Build and Test
         run: |
           dotnet build DfE.FindInformationAcademiesTrusts -c Release
-          dotnet coverage collect "dotnet test" -f xml -o "coverage.xml"
-          reportgenerator -reports:coverage.xml -targetdir:coverage
+          dotnet test --no-build --verbosity normal
```

```suggestion
          dotnet coverage collect "dotnet test" -f xml -o "coverage.xml"
```
We do need to collect coverage for SonarQube. The coverage collector is installed by `dotnet tool restore`